build-tools/packages/build-cli/src/commands/generate/changeset.ts

```diff
@@ -47,16 +45,16 @@ interface Choice {
 }
 
 export default class GenerateChangesetCommand extends BaseCommand<typeof GenerateChangesetCommand> {
-	static summary = `Generates a new changeset file. You will be prompted to select the packages affected by this change. You can also create an empty changeset to include with this change that can be updated later.`;
-	static aliases: string[] = [
+	static readonly summary = `Generates a new changeset file. You will be prompted to select the packages affected by this change. You can also create an empty changeset to include with this change that can be updated later.`;
+	static readonly aliases: string[] = [
```

Comment:
```suggestion
	static readonly aliases: readonly string[] = [
```

Reply:
This change also causes a compilation error because of the base class (I think).